redis/redis.go

```diff
@@ -27,11 +27,11 @@ import (
 	"fmt"
 	"time"
 
-	"github.com/bsm/redis-lock"
+	lock "github.com/bsm/redis-lock"
```

Comment:
I guess this change is to make the name more consistent, right?

Reply:
Yes, it is right!